.github/workflows/ci.yaml

```
@@ -58,6 +59,14 @@ jobs:
               - crates/ruff_dev/**
               - scripts/*
 
+            code:
+              - crates/*
+              - scripts/*
+              - Cargo.toml
+              - Cargo.lock
+              - clippy.toml
+              - pyproject.toml
```

```suggestion
              - pyproject.toml
              - rust-toolchain.toml
              - fuzz/**
```

Does the plugin support a way to say: Only run when e.g. all changed files are markdown files? It would reduce that we missed adding an entry. I rather have the job runs too often that miss out.

---

As I see no, it could not.  

But it could be addressed by running them on master, after the merge.

---

Why not with something like

```
- **/*
- !**/*.md
```

I would be surprised if this was not feasible.